src/System.Text.Json/tests/Serialization/EnumConverterTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
```

Comment:
Please add a test failure case that just uses `Enum` class as a property or return value from root (should fail).

Reply:
Not sure what you mean here. Use System.Enum directly?

Reply:
Yes something like `JsonSerializer.Parse<Enum>("1")` should fail gracefully (JsonException). I see we actually have a test for that -- so nevermind.